packages/js-commons/src/core/transporter.ts

```diff
@@ -3,6 +3,21 @@ import type { ApplicationDTO } from "./dto";
 export interface ApplicationTransporter {
   get: (appId: string) => Promise<ApplicationDTO>;
   put: (app: ApplicationDTO, userId: string) => Promise<ApplicationDTO>;
+  storeAsset: (
```

Comment:
These don't need to part of the interface they can just be exported as utilities.